command/server/config.go

```diff
@@ -60,8 +60,10 @@ func DevConfig(ha, transactional bool) *Config {
 			&Listener{
 				Type: "tcp",
 				Config: map[string]interface{}{
-					"address":     "127.0.0.1:8200",
-					"tls_disable": "1",
+					"address":                      "127.0.0.1:8200",
+					"tls_disable":                  true,
+					"proxy_protocol":               "use_if_authorized",
```

Comment:
I think the answer is "yes" although we don't do that with tls, so I worry it would be sort of a "okay, why are they changing this up now" issue UX wise. We could support that later on and move tls and proxy both.